docpages/make_a_bot/replit.md

```diff
@@ -1,8 +1,8 @@
-\page building-a-cpp-discord-bot-in-repl Creating a Discord bot in Repl.it
+\page building-a-cpp-discord-bot-in-repl Creating a Discord bot in Replit
```

Comment:
does the name need to be changed?

Reply:
Yes, it is no longer repl.it, but rather Replit.com